server/index.js

```diff
@@ -32,6 +32,25 @@ app.get('/api/products', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products/:productId', (req, res, next) => {
+  const { productId } = req.params;
+  const sql = `
+    SELECT *
+    FROM "products"
+    WHERE "productId" = ${productId}`;
```

Comment:
This query is vulnerable to SQL injection, use a Parameterized Query (see the `node-postgres` docs). The `productId` should be substituted in by Postgres, not in a template literal substitution.

Reply:
Before you query the database, be sure to validate the `productId` to make sure that it's a positive integer.